templates/docs/patterns/split-list/index.md

```diff
@@ -0,0 +1,133 @@
+| `is_list_full_width_on_tablet`         | boolean | `true`  | Whether the list items should appear full-width on their own rows on tablet          |
+
+### Slots
+
+| Name                                                   | Description                                                                                           |
+| ------------------------------------------------------ | ----------------------------------------------------------------------------------------------------- |
+| `title`                                                | HTML to be rendered as the Split List title                                                           |
+| `description`                                          | HTML to be rendered as the Split List description; if omitted, `title` will always display full-width |
+| `list_item_title_1` - `list_item_title_10`             | HTML to be rendered for each list item's title; max of 10                                             |
+| `list_item_description_1` - `list_item_description_10` | HTML to be rendered for each list item's description; max of 10                                       |
+
+<div class="p-notification--information">
+  <p class="p-notification__content">
+    <span class="p-notification__title">Note:</span>
+    <span class="p-notification__message">The following examples display rendered output HTML for reference purposes, but the Macro should be used whenever possible - it's easier to use!</span>
+  </p>
+</div>
+
+## 50/50 on desktop w/ description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-desktop-with-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## 50/50 on tablet w/o description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-tablet-no-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## 50/50 on tablet w/ description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-tablet-with-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## 50/50 w/ description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/50-50-with-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## Full-width w/o description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/full-width-no-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## Full-width w/ description
+
+<div class="embedded-example"><a href="/docs/examples/patterns/split-list/full-width-with-description/" class="js-example">
+View example of the matrix pattern
+</a></div>
+
+## Import
+
+To import and use this Macro in your project, follow the steps on the
+[Macro installation guide](TODO).
```

Comment:
If there is no installation guide yet, let's not document it as such.

At this point it technically doesn't matter if it's a macro or not (if devs can't use it). We also don't really have a practice of documenting these macros, so need to figure that out.

I'd just suggest documenting it as a standard HTML for now, with examples to copy (internally using macros of course). And we can build on top of that when we have means to install and document those macros.

Reply:
Also, this link is 404, causing linkchecker to fail.